Durable Functions documentation for GetStatus Paging HTTP APIs (#18599)
Azure/azure-content: articles/azure-functions/durable-functions-http-api.md

| createdTimeFrom  | Query string    | Optional parameter. When specified, filters the list of returned instances which were created at or after the given ISO8601 timestamp.|
| createdTimeTo    | Query string    | Optional parameter. When specified, filters the list of returned instances which were created at or before the given ISO8601 timestamp.|
| runtimeStatus    | Query string    | Optional parameter. When specified, filters the list of returned instances based on their runtime status. To see the list of possible runtime status values, see the [Querying instances](durable-functions-instance-management.md) topic. |

> | top    | Query string    | Optional parameter. When specified, split the results of the query into pages and limit the maximum number of data per page. |

Reviewer: ```suggestion
| top    | Query string    | Optional parameter. When specified, split the results of the query into pages and limit the maximum number of results per page. |
```

GET /runtime/webhooks/durableTask/instances/?taskHub={taskHub}&connection={connection}&code={systemKey}&createdTimeFrom={createdTimeFrom}&createdTimeTo={createdTimeTo}&runtimeStatus={runtimeStatus,runtimeStatus,...}
```

#### Request with paging

You can set the `top` parameter to split the results of the query into pages.

For Functions 1.0, the request format is as follows:

```http

> GET /admin/extensions/DurableTaskExtension/instances/?taskHub={taskHub}&connection={connection}&code={systemKey}&createdTimeFrom={createdTimeFrom}&createdTimeTo={createdTimeTo}&runtimeStatus={runtimeStatus,runtimeStatus,...}&top={top}

Reviewer: Let's simplify this example by removing the filters.
```suggestion
GET /admin/extensions/DurableTaskExtension/instances/?taskHub={taskHub}&connection={connection}&code={systemKey}&top={top}
```

```

The Functions 2.0 format has all the same parameters but a slightly different URL prefix: 

```http
GET /runtime/webhooks/durableTask/instances/?taskHub={taskHub}&connection={connection}&code={systemKey}&createdTimeFrom={createdTimeFrom}&createdTimeTo={createdTimeTo}&runtimeStatus={runtimeStatus,runtimeStatus,...}&top={top}

> If the next page exists, continuation token is returned in the response header.  This key is `x-ms-continuation-token`.

Reviewer: ```suggestion
If a next page exists, a continuation token is returned in the response header named `x-ms-continuation-token`.
```

> If you set continuation token value in the next request header, you can get the next page.  This key in request header is `x-ms-continuation-token`.

Reviewer: ```suggestion
If you set the returned `x-ms-continuation-token` header in the next status query request, you will get the next page.
```
